packages/loader/container-loader/src/createAndLoadContainerUtils.ts

```diff
@@ -253,6 +253,23 @@ export async function loadFrozenContainerFromPendingState(
  */
 export async function loadSummarizerContainerAndMakeSummary(
 	loadSummarizerContainerProps: ILoadSummarizerContainerProps,
+): Promise<LoadSummarizerSummaryResult> {
+	let result = await loadSummarizerContainerAndMakeSummaryInternal(
+		loadSummarizerContainerProps,
+	);
+	if (!result.success) {
```

Comment:
Should we also check something about the error?  e.g. `canRetry` (and then make sure that's set properly in the specific case you ran into).

Something to consider if we're working here again.  I can't think of a risk of doing 2 shots in all cases, so seems ok to leave for now.